docs/scripts/buildApi.ts

```diff
@@ -284,9 +284,9 @@ async function updateStylesDefinition(context: {
             .types;
 
           if (types) {
-            types.forEach((node) => {
-              if (node.type === 'TSTypeLiteral') {
-                classesDeclarationNode = node;
+            types.forEach((n) => {
```

Comment:
Could we keep them as was? It's more descriptive.

Reply:
Lint was complaining as node already exists as the name in the upper scope (if you are talking about vars names)

Reply:
I was, but okay